src/less/index.less

```diff
@@ -1,34 +1,320 @@
+        color: #FF914C;
+    }
+
+    &-FF914D {
+        color: #FF914D;
+    }
+
+    &-4C4C49 {
+        color: #4C4C49;
+    }
+
+    &-549BEA {
+        color: #549BEA;
+    }
+
+    &-blue {
+        color: blue;
+    }
+
+    &-white {
+        color: white;
+    }
+}
+
+.center {
+    text-align: center;
+}
+
+.line-height-0_5 {
+    line-height: 0.5;
+}
+
+.margin {
+    &-0 {
+        margin: 0;
+    }
+
+    &-0-100 {
+        margin: 0 100px;
+    }
+
+    &-5 {
+        margin: 5px;
+    }
+
+    &-5vw {
+        margin: 5vw;
+    }
+
+    &-10 {
+        margin: 10px;
+    }
+
+    &-16-0 {
+        margin: 16px 0;
+    }
+
+    &-1rem {
+        margin: 1rem;
+    }
+
+    &-1rem-0 {
+        margin: 1rem 0;
+    }
+
+    &-1rem-0-2rem-0 {
+        margin: 1rem 0 2rem 0;
+    }
+
+    &-top {
+        &-0 {
+            margin-top: 0;
+        }
+
+        &-10 {
+            margin-top: 10px;
+        }
+
+        &-15 {
+            margin-top: 15px;
+        }
+
+        &-200 {
+            margin-top: 200px;
+        }
+    }
+
+    &-bottom {
+        &-0 {
+            margin-bottom: 0;
+        }
+
+        &-25 {
+            margin-bottom: 25px;
+        }
+
+        &-50 {
+            margin-bottom: 50px;
+        }
+    }
+
+    &-left {
+        &-0 {
+            margin-left: 0;
+        }
+
+        &-25 {
+            margin-left: 25px;
+        }
+    }
+
+    &-right {
+        &-0 {
+            margin-right: 0;
+        }
+
+        &-10 {
+            margin-right: 10px;
+        }
+    }
+}
+
+.padding {
+    &-0-10 {
+        padding: 0 10px;
+    }
+
+    &-0-0-25-25 {
+        padding: 0 0 25px 25px;
+    }
+
+    &-10 {
+        padding: 10px;
+    }
+
+    &-10-0 {
+        padding: 10px 0;
+    }
+
+    &-24 {
+        padding: 24;
+    }
+
+    &-25-0 {
+        padding: 25px 0;
+    }
+
+    &-35 {
+        padding: 35px;
+    }
+
+    &-50 {
+        padding: 50px;
+    }
+
+    // &-top {}
+    &-bottom {
+        &--10 {
+            padding-bottom: -10px;
+        }
+
+        &-25 {
+            padding-bottom: 25px;
+        }
+    }
+
+    &-left {
+        &-20pct {
+            padding-left: 20%;
+        }
+
+        &-50 {
+            padding-left: 50px;
+        }
+    }
+
+    // &-right {}
+}
+
+.border {
+    &-1-solid-ccc {
+        border: 1px solid #ccc;
+    }
+
+    &-radius {
+        &-5 {
+            border-radius: 5px;
+        }
+
+        &-10 {
+            border-radius: 10px;
+        }
+
+        &-30 {
+            border-radius: 30px;
+        }
+    }
+}
+
+.width {
+    &-80 {
+        width: 80%;
+    }
+
+    &-100 {
+        width: 100%;
+    }
+
+    &-600 {
+        width: 600px;
+    }
+}
+
+.height {
+    &-20vh {
+        height: 20vh;
+    }
+
+    &-200 {
+        height: 200px;
+    }
+}
+
+.min {
+    &-height {
+        &-380 {
+            min-height: 380;
+        }
+    }
+}
+
+.max {
+    &-height {
+        &-40vh {
+            max-height: 40hv;
```

Comment:
just need to flip the units. `vh` instead of `hv`.

Reply:
Derp; thanks @Brodt7258!